serde-generate/runtime/golang/lcs/deserializer.go

```diff
@@ -181,12 +179,14 @@ func (d *Deserializer) DeserializeOptionTag() (bool, error) {
 }
 
 func (d *Deserializer) GetBufferOffset() uint64 {
-	return d.initial_size - uint64(d.buf.Len())
+	return uint64(len(d.input)) - uint64(d.buf.Len())
```

Comment:
the input length grows?

Reply:
We need to keep the input byte vector `d.input` for checking key map order efficiently.